packages/babel-plugin-transform-classes/src/transformClass.ts

```diff
@@ -600,8 +602,26 @@ export default function transformClass(
     );
   }
 
+  function extractDynamicKeys() {
+    const { dynamicKeys, node, scope } = classState as {
+      dynamicKeys: Map<string, t.Expression>;
+      node: t.Class;
+      scope: NodePath["scope"];
+    };
+
+    node.body.body.forEach(elem => {
```

Comment:
Foreach performance is approximately 6 times slower than FOR / FOREACH performance. The FOR loop without length caching works 3 times slower on lists, comparing to arrays.

For those who are interested in performance - ForEach is 96% slower than for loop..

https://stackoverflow.com/questions/43821759/why-array-foreach-is-slower-than-for-loop-in-javascript